src/Kentico.Xperience.K13Ecommerce/Synchronization/ContentItems/ContentItemFolderSynchronizationService.cs

```diff
@@ -36,22 +36,33 @@ public async Task SynchronizeContentItemFolders(CancellationToken cancellationTo
             return;
         }
 
-        await SynchronizeProductFolder(settings.K13EcommerceSettingsProductSKUFolderID, ProductSKU.CONTENT_TYPE_NAME, cancellationToken);
-        await SynchronizeProductFolder(settings.K13EcommerceSettingsProductVariantFolderID, ProductVariant.CONTENT_TYPE_NAME, cancellationToken);
-        await SynchronizeProductFolder(settings.K13EcommerceSettingsProductImageFolderID, ProductImage.CONTENT_TYPE_NAME, cancellationToken);
+        await SynchronizeProductFolder(settings.K13EcommerceSettingsProductSKUFolderGuid, ProductSKU.CONTENT_TYPE_NAME, cancellationToken);
+        await SynchronizeProductFolder(settings.K13EcommerceSettingsProductVariantFolderGuid, ProductVariant.CONTENT_TYPE_NAME, cancellationToken);
+        await SynchronizeProductFolder(settings.K13EcommerceSettingsProductImageFolderGuid, ProductImage.CONTENT_TYPE_NAME, cancellationToken);
     }
 
-    private async Task SynchronizeProductFolder(int folderId, string contentTypeName, CancellationToken cancellationToken = default)
+    private async Task SynchronizeProductFolder(Guid targetFolderGuid, string contentTypeName, CancellationToken cancellationToken = default)
     {
-        if (folderId == 0)
+        if (targetFolderGuid == Guid.Empty)
         {
             return;
         }
 
+        var folder = await contentFolderManager.Get(targetFolderGuid, cancellationToken);
+        if (folder == null)
```

Comment:
According to our code-style conventions we put empty lines around `if-else` blocks.

Reply:
Empty lines added.